packages/react/src/components/DataTable/TableExpandHeader.js

```diff
@@ -49,6 +49,21 @@ const TableExpandHeader = ({
   );
 };
 
+function isRequiredIfPropExists(requiredProp) {
```

Comment:
Just wanted to see if you want to re-use: https://github.com/carbon-design-system/carbon/blob/v10.10.0/packages/react/src/prop-types/requiredIfGivenPropExists.js

Reply:
🤦‍♂ yes, I'll definitely refactor this to use the existing function!